extensions/ql-vscode/src/definitions.ts

```diff
@@ -34,14 +34,24 @@ function tagOfKeyType(keyType: KeyType): string {
   }
 }
 
+function nameOfKeyType(keyType: KeyType): string {
+  switch (keyType) {
+    case KeyType.DefinitionQuery: return "definitions";
+    case KeyType.ReferenceQuery: return "references";
+  }
+}
+
 async function resolveQueries(cli: CodeQLCliServer, qlpack: string, keyType: KeyType): Promise<string[]> {
   const suiteFile = tmp.fileSync({ postfix: '.qls' }).name;
   const suiteYaml = { qlpack, include: { kind: 'definitions', 'tags contain': tagOfKeyType(keyType) } };
   await fs.writeFile(suiteFile, yaml.safeDump(suiteYaml), 'utf8');
 
   const queries = await cli.resolveQueriesInSuite(suiteFile, helpers.getOnDiskWorkspaceFolders());
   if (queries.length === 0) {
-    throw new Error("Couldn't find any queries for qlpack");
+    vscode.window.showErrorMessage(
+      `No ${nameOfKeyType(keyType)} queries (tagged "${tagOfKeyType(keyType)}") could be found in the current library path. It might be necessary to upgrade the CodeQL libraries.`
```

Comment:
This is fine, though I wonder if the message could link out to some instructions on how to do that. Not really necessary for this to go through.